static/templates/admin/editStreamingPropertiesForm.html

```diff
@@ -1,8 +1,13 @@
 <iframe id="upload_file_frame" name="upload_file_frame" height="0" width="0" style="display: none;"></iframe>
 <form id="edit_streaming_properties" class="edit_form edit_streaming_properties_form">
+    <p class="form_notes">
+        Structure: https://durastream.lib.unc.edu/player?spaceId=[space-id]&filename=[filename-without-extension]
+    <br />
+        Example: https://durastream.lib.unc.edu/player?spaceId=open-hls&filename=04950_VT0008_0001
+    </p>
```

Comment:
I think it might make more sense to put this below in the input rather than before, or at least that's where I'm used to explanatory text for a field being.